Framework/CurveFitting/src/Algorithms/EstimateFitParameters.cpp

```diff
@@ -102,6 +106,39 @@ void fixBadParameters(CostFunctions::CostFuncFitting &costFunction,
   }
 }
 
+/// A class for sorting and storing sets of best function parameters.
+class BestParameters {
+  /// Maximum size of the store
+  size_t m_size;
+  /// Actual storage.
+  std::map<double, GSLVector> m_params;
```

Comment:
At  this moment I think it is important. It helps select the `n` best values and find the worst of the best which has to be thrown away after a new best is found. The user may also want to know which is the best, which is the second best and so on.